README.md

```diff
@@ -306,22 +306,31 @@ task.plugins {
 
 #### Default outputs
 
-**Java** projects: the `java` builtin is added by default.  If you
-wish to remove this output, for example, to only generate for Python:
+**Java** projects: the `java` builtin is added by default: without any further specification, Java classes will be generated during the build process.
+
+**Python** output can be generated by adding the `python` builtin:
 
 ```gradle
 protobuf {
+  generatedFilesDir = "$projectDir/generated"
+
   generateProtoTasks {
     all().each { task ->
       task.builtins {
-        remove java
+        // Generates Python code in the output folder:
         python { }
+
+        // If you wish to avoid generating Java files:
+        remove java
       }
     }
   }
 }
 ```
 
+See [this section](#change_where_files_are_generated) for details about where the code will be generated.
```

Comment:
The anchor name generated by github uses dashes instead of underscores.

Reply:
oh, dear - you'd hope they'd all agree on a common standard.
Fixed, sorry about that!